API/NoDocs/Microsoft.Extensions.Diagnostics.ResourceMonitoring/LinuxResourceUtilizationCounters.cs

```diff
@@ -0,0 +1,11 @@
+// Assembly 'Microsoft.Extensions.Diagnostics.ResourceMonitoring'
+
+using System.Runtime.CompilerServices;
+
+namespace Microsoft.Extensions.Diagnostics.ResourceMonitoring;
+
+public static class LinuxResourceUtilizationCounters
+{
+    public static string CpuConsumptionPercentage { get; }
+    public static string MemoryConsumptionPercentage { get; }
```

Comment:
Do these need to be public? How would someone use them?

Reply:
ObservableInstruments.

Reply:
Fwiw, we haven't provided public string constants for the names of our instruments anywhere else that we added them. My assumption would be that:
- if you are consuming from inside the process in production, APIs are a better way to expose the data than extracting it from instruments.
- if you are writing unit tests, hard-code the name in the test so that if it ever changes the test fails signaling a breaking change has occured
- if you are consuming from outside the process the property won't be available and the name would need to be hard-coded regardless

If it were my choice I wouldn't make them public, but on a prioritized list of issues my level of concern here is low.